apps/api/src/app/subscribers/e2e/get-preferences.e2e.ts

```diff
@@ -16,13 +17,44 @@ describe('Get Subscribers preferences - /subscribers/preferences/:subscriberId (
     });
   });
 
-  it('should get subscriber preferences', async function () {
-    const response = await getPreference(session);
+  it('should get subscriber workflow preferences with inactive channels by default', async function () {
+    const response = await getPreference(session, session.subscriberId);
+    const data = response.data.data[0];
+
+    expect(data.preference.channels).to.deep.equal({
+      email: true,
+      in_app: true,
+      push: true,
+      chat: true,
+      sms: true,
+    });
+  });
+
+  it('should get subscriber workflow preferences with inactive channels when includeInactiveChannels is true', async function () {
+    const response = await getPreference(session, session.subscriberId, {
+      includeInactiveChannels: true,
+    });
+    const data = response.data.data[0];
+
+    expect(data.preference.channels).to.deep.equal({
```

Comment:
Shouldn't we see a difference in the returned channels between this test and the previous one since the `includeInactiveChannels` flag is included?

Reply:
Yes we do expect different behaviour, but for the following test instead. The default flag value is `true`, so these first two tests are expected to have the same behaviour.

See [this test below](https://github.com/novuhq/novu/pull/7101/files/c2001924b30d04facd0346e79108aad86be446c6#diff-c41eb3bd24095c41b35226dc58ce7c641d6f3c25d679c92dd151952c5ff03c30R48) for `includeInactiveChannels: false` which has some channels omitted.